builder-18.toml

```diff
@@ -8,36 +8,36 @@ version = "0.10.1"
 
 [[buildpacks]]
   id = "heroku/maven"
-  uri = "https://github.com/heroku/heroku-buildpack-java/releases/download/v68/heroku-java-cnb-v68.tgz"
+  uri = "docker://public.ecr.aws/r2f9u0w4/heroku-maven-buildpack:0.2.0"
 
 [[buildpacks]]
   id = "heroku/jvm"
-  uri = "https://github.com/heroku/heroku-buildpack-jvm-common/releases/download/v111/heroku-jvm-common-cnb-v111.tgz"
+  uri = "docker://public.ecr.aws/r2f9u0w4/heroku-jvm-buildpack:0.1.2"
 
 [[buildpacks]]
   id = "heroku/java"
-  uri = "buildpacks/heroku_java"
+  uri = "docker://public.ecr.aws/r2f9u0w4/heroku-java-buildpack:0.1.3"
+
+[[buildpacks]]
+  id = "heroku/gradle"
+  uri = "docker://public.ecr.aws/r2f9u0w4/heroku-gradle-buildpack:0.0.34"
```

Comment:
Do you have a versioning plan if you need to cut a new release of the same base version but with an updated shim? We were discussing moving all the shimmed buildpacks to a registry like this. Will you just overwrite `0.0.34` or bump the version in some way?

Reply:
Good point. There is no versioning plan for this. I think we would just overwrite should that case ever arise. The chance of this happening is quite slim I think. And if it were to become a problem, we can always change the versioning scheme it so smth. like `1.${V2_VERSION}.${SHIM_VERSION}` without violating semver.

Also, at least for the JVM buildpacks, I'm trying to move away from shimmed V2 buildpacks ASAP. Happy to change the versioning scheme if you believe it to be a problem though.